README.md

```diff
@@ -20,6 +22,17 @@ Have something to add or change? Open a [pull request](https://github.com/luqmaa
 - [GTFS-realtime](#gtfs-realtime)
 - [SIRI](#siri)
 
+### Community
+
+Places to ask questions and find other community resources.
+
+- [TransitWiki](http://transitwiki.org) - 
+- [Transit Developers Mailing List](https://groups.google.com/forum/#!forum/transit-developers)
+- [OneBusAway Slack](https://onebusaway.slack.com) - Slack chat for the OneBusAway project.
```

Comment:
I'd change this URL to https://onebusaway.herokuapp.com/ - that's where new users can sign themselves up.